README.md

```diff
@@ -47,7 +70,7 @@ to your preferred location before running the `zeppelin.sh --start` script
 ### Zeppelin interpreter memory
 
 By default half of the total available memory will be allocated to the Zeppelin interpreters on start.
-You can override this value by setting the environment variable `ZEPPELIN_INTP_MEMORY` (in Gb, eg: `export ZEPPELIN_INTP_MEMORY=8` for 8 Gb of memory)
+You can override this value by setting the environment variable `ZEPPELIN_INTP_MEMORY` (the value should be `-Xmx<memory>g`, where `<memory>` is the size in GB, eg: `export ZEPPELIN_INTP_MEMORY="-Xmx8g"` for 8 Gb of memory).
```

Comment:
Actually this is not correct. You need to specify the memory in Go as an int (see `env.sh`: `export ZEPPELIN_INTP_MEMORY=${ZEPPELIN_INTP_MEMORY:--Xmx${INTEPRETER_MEMORY}g}`). The "-Xmx" and "g" prefix and suffix will be added be the script :)

Reply:
From what I recall, 

    export ZEPPELIN_INTP_MEMORY=${ZEPPELIN_INTP_MEMORY:--Xmx${INTEPRETER_MEMORY}g}

means either `ZEPPELIN_INTP_MEMORY` is already defined (and thus does not change), or it is set to `-Xmx${INTEPRETER_MEMORY}g`. So the `-Xmx` is never added by the script ! 

Just to be sure, I ran:
```
export ZEPPELIN_INTP_MEMORY=4
./zeppelin.sh --start
echo $ZEPPELIN_INTP_MEMORY # gives 4
```
Zeppelin starts but I cannot run any cell because it passes `4` as startup option.

Reply:
Ah! You are completely correct my bad! What do you think about changing it to
```
export ZEPPELIN_INTP_MEMORY="-Xmx ${ZEPPELIN_INTP_MEMORY:-${INTEPRETER_MEMORY}}g"
``` ? 

This way the user never has to care about the Xmx flag

Reply:
I like the ides, but I would suggest using another variable to avoid the case of the script running twice in the same shell... we would end up with `-Xmx-Xmx4gg` :(.

Reply:
Good catch!